README.md

```diff
@@ -10,7 +10,36 @@ This repository contains a GitHub action that simplifies the usage of the [xrefc
 
 ## Usage
 
-TODO
+By default this action installs the latest version of `xrefcheck` (it downloads a static binary from GitHub releases) and runs it with no arguments.
+
+### Inputs
+
+Please refer to [action.yml](./action.yml) for a full list of inputs.
+Quick overview:
+* `xrefcheck-version` lets you pick a specific version.
+* `xrefcheck-args` lets you pass custom arguments.
+* `action` can be used if just want to install the tool.
+
+### Outputs
+
+`xrefcheck-path` output will contain a relative path to the `xrefcheck` executable.
+
+### Example
+
+To run `xrefcheck-0.1.1` on your repository without progress reporting (since it doesn't look nice on CI):
```

Comment:
It seems to me that we always want `--no-progress` to be there, so it can be part of the action itself.
If that makes sense, than as an example you can say that one can check only local refs with `-m local-only`

Reply:
That's true, but progress reporting on CI looks more or less sanely and maybe some people will want to enable it :man_shrugging: 

I think this action (i. e. `xrefcheck-action`) can/should make decisions about reasonable defaults (that is disable reporting), but shouldn't completely disable reporting (or another feature) without making it possible to enable it. To clarify: it's hard to imagine that progress reporting can be useful inside GitHub Actions, but at the same time it's also hard to imagine all use cases for this action (at this stage it seems pretty straightforward, but software and its usage scenarios tend to get bigger over time).

Since this tool is supposed to be primarily used on CI, I would suggest disabling progress reporting by default and adding `--progress` or `--report-progress` flag. Or maybe not disabling by default but having both `--progress` and `--no-progress` and giving precedence to the last one (AFAIK some tools do that). What do you think?

Another reason why I don't want to hardcode any command line option here is to keep this action more resistant to `xrefcheck` CLI changes. That's a minor reason, but probably it still counts.

![image](https://user-images.githubusercontent.com/2556409/85466019-23580a80-b5b2-11ea-8828-49d841dafa3b.png)

Reply:
Okay, sounds fair.

Also if progress bar works like on picture, then probably it makes sense to have for long timeouts.

Reply:
You can go to the "Checks" tab in this PR to see how it all looks :)

Reply:
Ah right, I first thought checks won't be enabled until merged to master, nice they they already are.

Reply:
I have `uses: ./`, it uses the action present right in the PR branch. So if you later open another PR and break something, it will be caught by CI immediately.

Reply:
Opened https://github.com/serokell/xrefcheck/issues/42

I guess I'll do it myself soon. Then they only change required in this PR will be README update.